src/common/util.js

```diff
@@ -561,6 +561,7 @@ function addCommentHelper(node, comment) {
   const comments = node.comments || (node.comments = []);
   comments.push(comment);
   comment.printed = false;
+  comment.nodeDescription = describeNodeForDebugging(node);
```

Comment:
Do you think it better be a getter?

Reply:
It's difficult for me to evaluate the impact and understand which way is better. Strings at least don't hold references to the node.